src/test/ui/resolve/privacy-enum-ctor.stderr

```diff
@@ -67,11 +90,16 @@ error[E0423]: expected value, found enum `E`
 LL |     let _: E = E;
    |                ^
    |
-   = help: you might have meant to use one of the enum's other variants that have fields
-help: try using one of the enum's variants
+help: try using the enum's variant which have fields
    |
 LL |     let _: E = E::Unit;
    |                ^^^^^^^
+help: try using one of the enum's variants which have fields
+   |
+LL |     let _: E = (E::Fn(/* fields */));
+   |                ^^^^^^^^^^^^^^^^^^^^^
+LL |     let _: E = (E::Struct { /* fields */ });
+   |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
What about

```
help: you might have meant to use the following enum variant(s)
  |
help: alternatively, the following enum variants are also available
  |
```

Reply:
I've updated the PR to use this wording.